xml/Microsoft.Win32/Registry.xml

```diff
@@ -288,29 +288,8 @@ The <xref:Microsoft.Win32.Registry> class also contains `static` <xref:Microsoft
       </ReturnValue>
       <Docs>
         <summary>Contains dynamic registry data. This field reads the Windows registry base key HKEY_DYN_DATA.</summary>
-        <remarks>
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
- The Windows 98/Windows Me registry supports both static data (which is stored on disk in the registry) and dynamic data (which changes frequently, such as performance statistics). This dynamic data area is the mechanism that allows Virtual Device Drivers (VxDs) to provide real-time data to Win32 applications that can run remotely as well as locally. It also allows the system monitor to provide performance statistics on remote Windows 98/Windows Me systems.
-
- VxDs are not limited to performance data. They can provide any data they want to pass from Ring 0 to Ring 3 efficiently without monopolizing the CPU. The registry supports dynamic data by storing a pointer to a function that returns a value (or many values). When a Registry call queries values associated with a dynamic key, that function is called to return the desired value or values.
-
-> [!NOTE]
->  Dynamic keys were introduced in Microsoft Windows 95 to handle dynamic Registry data. They are supported only in Windows 98/Windows Me.
-
-
-
-## Examples
- The following example demonstrates how to retrieve the subkeys of this key, and prints their names to the screen. Use the <xref:Microsoft.Win32.RegistryKey.OpenSubKey%2A> method to create an instance of the particular subkey of interest. You can then use other operations in <xref:Microsoft.Win32.RegistryKey> to manipulate that key. Note that this example can return no results, since there might not be dynamic data available, or you might not be running Windows 98/ME. Using this key may cause an error on other systems.
-
- [!code-cpp[Classic Registry.DynData Example#1](~/samples/snippets/cpp/VS_Snippets_CLR_Classic/classic Registry.DynData Example/CPP/source.cpp#1)]
- [!code-csharp[Classic Registry.DynData Example#1](~/samples/snippets/csharp/VS_Snippets_CLR_Classic/classic Registry.DynData Example/CS/source.cs#1)]
- [!code-vb[Classic Registry.DynData Example#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_Classic/classic Registry.DynData Example/VB/source.vb#1)]
-
- ]]></format>
-        </remarks>
-        <exception cref="T:System.ObjectDisposedException">The operating system does not support dynamic data; that is, it is not Windows 98, Windows 98 Second Edition, or Windows Millennium Edition (Windows Me).</exception>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ObjectDisposedException">The operating system does not support dynamic data; that is, it is not Windows 98 or Windows Millennium Edition (Windows Me).</exception>
```

Comment:
I don't think this field is supported any more. It no-ops in Desktop CLR (since Desktop CLR doesn't target 9x / ME), and it doesn't exist at all in Core.

Reply:
In other words, are you suggesting to remove the exception?

```suggestion
```

Reply:
Yes either remove it entirely or note that the API exists only on .NET Framework and even there it does nothing.